jest.config.mjs

```diff
@@ -77,5 +80,11 @@ module.exports = {
     'jest-extended/all',
     '<rootDir>/tests/setupAfterEnv.ts'
   ],
-  moduleNameMapper: moduleNameMapper,
+  extensionsToTreatAsEsm: ['.ts'],
+  moduleNameMapper: {
+    ...moduleNameMapper,
+    "^(\\.{1,2}/.*)\\.js$": "$1",
```

Comment:
This one is not mentioned in the docs. It appears to basically remove the `.js` extension and load things using extensionless imports. I'm not entirely sure why this is necessary. But it is. Otherwise things just don't work and no importing is possible.